front/app/modules/commercial/content_builder/admin/components/ContentBuilderToolbox/index.tsx

```diff
@@ -3,31 +3,99 @@ import React from 'react';
 // craft
 import { useEditor, Element } from '@craftjs/core';
 import Container from '../CraftComponents/Container';
+import Text from '../CraftComponents/Text';
 
 // intl
 import { injectIntl } from 'utils/cl-intl';
 import { InjectedIntlProps } from 'react-intl';
 import messages from '../../messages';
-import { Box } from '@citizenlab/cl2-component-library';
+import { Box, Icon } from '@citizenlab/cl2-component-library';
+
+// Styling
+import styled from 'styled-components';
+import { colors } from 'utils/styleUtils';
+
+const ToolboxSection = styled(Box)`
+  border-style: solid;
+  border-right-style: none;
+  border-top-style: none;
+  border-width: 1px;
+  border-color: ${colors.border};
+  width: 100%;
+`;
+
+const ComponentToolboxItem = styled(Box)`
+  margin-bottom: 16px;
+  width: 100%;
+  display: flex;
+  gap: 20px;
+  padding-left: 10px;
+`;
+
+const ToolboxItemText = styled.p`
+  color: ${colors.adminDarkestBackground};
+  font-size: 15px;
+`;
+
+const ToolboxHeader = styled.h1`
+  padding-top: 10px;
+  font-size: 16px;
+  padding-left: 10px;
+  color: ${colors.adminDarkestBackground};
+`;
+
+const ToolboxIcon = styled(Icon)`
+  width: 18px;
+  height: 18px;
+  fill: ${colors.clBlue};
+`;
 
 const ContentBuilderToolbox = ({
   intl: { formatMessage },
 }: InjectedIntlProps) => {
   const { connectors } = useEditor();
   return (
-    <Box marginTop="20px">
-      <button
-        ref={(ref) =>
-          ref &&
-          connectors.create(
-            ref,
-            <Element canvas is={Container} id="container" />
-          )
-        }
-      >
-        {formatMessage(messages.oneColumn)}
-      </button>
-    </Box>
+    <>
+      <ToolboxSection>
+        <ToolboxHeader>Sections</ToolboxHeader>
```

Comment:
We don't need these headers for now - or the separators between them - see Ben's comments in the prototype.